### PR TITLE
Added analytics HA docs

### PR DESCRIPTION
**Pull request by ruks**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+To configure API-M Analytics as a scalable deployment, you can use the Active-Active deployment pattern. For an 
+overview of the Active-Active deployment pattern and instructions to configure it, see the following topics.
+
+- [Overview](#overview)
+- [Configuring an active-active cluster](#configuring-an-active-active-cluster)
+
+
+## Overview
+![](../../../../assets/img/setup-and-install/configure_apim_analytics/Active-Active_Deployment.png)
```
Comment (Mariangela) [inline en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md]: Use {{base_path}} in the URLs without ../

Example:
{{base_path}} /assets/img/setup-and-install/configure_apim_analytics/Active-Active_Deployment.png

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+In order to perform aggregations in a distributed manner and achieve the scalability, this setup uses distributed aggregations.
+
+Distributed aggregations partially process aggregations in different nodes. This allows you to assign one node to 
+process only a part of an aggregation (regional scaling, etc.). In order to do this all the aggregations must have a 
+physical database and must be linked to the same database.
+
+Partitioning aggregations can be enabled at aggregation level and also at a global level. To enable it at the global 
+level, add the following section with the *PartitionById* annotation set to true in the 
+<APIM-ANALYTICS_WORKER_HOME>/conf/worker/deployment.yaml file.
```
Comment (Mariangela) [inline en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md]: File names and paths should be within ` `
`<APIM-ANALYTICS_WORKER_HOME>/conf/worker/deployment.yaml`

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md (hunk 1 above)
Comment (ruks) [reply en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md]: Fixed

File: en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md (hunk 2 above)
Comment (ruks) [reply en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md]: Fixed

**Pull request merged**